[2024-09-19T23:21:04.976+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion scheduled__2024-09-19T20:00:00+00:00 [queued]>
[2024-09-19T23:21:05.015+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion scheduled__2024-09-19T20:00:00+00:00 [queued]>
[2024-09-19T23:21:05.017+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 0
[2024-09-19T23:21:05.065+0000] {taskinstance.py:1382} INFO - Executing <Task(SSHOperator): raw_ingestion> on 2024-09-19 20:00:00+00:00
[2024-09-19T23:21:05.078+0000] {standard_task_runner.py:57} INFO - Started process 2170 to run task
[2024-09-19T23:21:05.079+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion', 'raw_ingestion', 'scheduled__2024-09-19T20:00:00+00:00', '--job-id', '211', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion.py', '--cfg-path', '/tmp/tmpny_ngx93']
[2024-09-19T23:21:05.081+0000] {standard_task_runner.py:85} INFO - Job 211: Subtask raw_ingestion
[2024-09-19T23:21:05.419+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion.raw_ingestion scheduled__2024-09-19T20:00:00+00:00 [running]> on host airflow-worker
[2024-09-19T23:21:06.269+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion' AIRFLOW_CTX_TASK_ID='raw_ingestion' AIRFLOW_CTX_EXECUTION_DATE='2024-09-19T20:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-19T20:00:00+00:00'
[2024-09-19T23:21:06.270+0000] {ssh.py:141} INFO - Creating ssh_client
[2024-09-19T23:21:06.270+0000] {ssh.py:114} INFO - ssh_hook is not provided or invalid. Trying ssh_conn_id to create SSHHook.
[2024-09-19T23:21:06.360+0000] {base.py:73} INFO - Using connection ID 'spark_ssh_default' for task execution.
[2024-09-19T23:21:06.362+0000] {ssh.py:300} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-09-19T23:21:06.418+0000] {transport.py:1893} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-09-19T23:21:07.011+0000] {transport.py:1893} INFO - Authentication (password) successful!
[2024-09-19T23:21:07.013+0000] {ssh.py:478} INFO - Running command: export SPARK_HOME=/usr/local/spark; /opt/conda/bin/spark-submit --master local[*] --executor-memory 2g /opt/shared/scripts/raw_ingestion.py
[2024-09-19T23:21:17.243+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 178, in execute
    result = self.run_ssh_client_command(ssh_client, self.command, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 163, in run_ssh_client_command
    exit_status, agg_stdout, agg_stderr = self.hook.exec_ssh_client_command(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/hooks/ssh.py", line 549, in exec_ssh_client_command
    raise AirflowException("SSH command timed out")
airflow.exceptions.AirflowException: SSH command timed out
[2024-09-19T23:21:17.248+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=raw_ingestion, task_id=raw_ingestion, execution_date=20240919T200000, start_date=20240919T232104, end_date=20240919T232117
[2024-09-19T23:21:17.300+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 211 for task raw_ingestion (SSH command timed out; 2170)
[2024-09-19T23:21:17.341+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-19T23:21:17.477+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
